Tutorials/start/rfplayground.md

```diff
@@ -89,5 +89,3 @@ Base stations use the APRS "#" symbol representing a "digi" as shown below on [a
 Your Faraday radio is now setup with a callsign and node ID it will use to identify itself even after reboot. We've also installed all necessary programs and configured them to communicate over USB serial ports and over RF using the 33cm ham band. You're ready to go!
```

Comment:
ACTION @kb1lqc : Line 47 you need to specify WHERE the use can find the telemtry.db file which has changed with recent updates to the Users .faraday folder.